compiler/lib/src/main/scala/codegen/CppWriter/CppDoc.scala

```diff
@@ -42,9 +42,14 @@ object CppDoc {
     superclassDecls: Option[String],
     /** The class members */
     members: List[Class.Member],
+    /** The class qualifier, if any, before the colon */
+    qualifier: Class.Qualifier = Class.NonFinal
```

Comment:
Can we call this qualifier `FinalQualifier` for consistency with `ConstQualifier`?

Reply:
Done!